bravado/mapping/operation.py

```diff
@@ -63,7 +75,7 @@ def build_params(self):
         param_specs = op_param_specs + path_param_specs
 
         for param_spec in param_specs:
-            param = Param(self.swagger_spec, param_spec)
+            param = Param(self.swagger_spec, self, param_spec)
```

Comment:
`operation_id` is also used. I predict `produces` will also be used at some point too.